plugins/serendipity_event_textile/serendipity_event_textile.php

```diff
@@ -251,7 +252,7 @@ function _process_tag_php($text)
                 '<font color="',
                 '</font>',
                 "\n ",
-                '� '
+                '� '
```

Comment:
unsure what this character would be (probably some ISO-8859-1 one?!) and what it would then maybe break. Could also be a UTF-8 BOM in ISO-8859-1?!